bin/liquidator/src/main.rs

```diff
@@ -234,15 +245,15 @@ async fn main() -> anyhow::Result<()> {
         forbidden_tokens: liq_config.forbidden_tokens.clone(),
     };
 
-    let mut rebalance_interval = tokio::time::interval(Duration::from_secs(30));
-    let (rebalance_trigger_sender, rebalance_trigger_receiver) = async_channel::bounded::<()>(1);
+    let (rebalance_trigger_sender, rebalance_trigger_receiver) = tokio::sync::mpsc::channel(1);
+    let (tx_trigger_sender, tx_trigger_receiver) = tokio::sync::mpsc::channel::<TxTrigger>(100);
```

Comment:
This is where I worry about the usual bounded channel semantic. We only care about the most recent, but this channel will block new generations when it's full. 

At some point we should write (a wrapper around an existing?) channel that drops the oldest entry when a new one is pushed to it. For now it can be done receiver-side, but will need to check that.

Just putting triggers in the channel makes it a lot less problematic, but still: could be full of "tcs" triggers and newer liquidation triggers can't be added.

Is it even helpful to have typed triggers? The only thing you want is to wake up a tx_sender thread, so it can pick up some available work if it was sleeping?